var/spack/repos/builtin/packages/snakemake/package.py

```diff
@@ -130,5 +130,6 @@ class Snakemake(PythonPackage):
     )
     depends_on("py-requests", when="+http", type=("build", "run"))
 
-    def test(self):
+    def test_run(self):
+        """Test if snakemake runs correctly"""
         Executable("snakemake")("--version")
```

Comment:
Since you're here, it would be prudent to ensure the test is using the installed executable:

```suggestion
        Executable(self.prefix.bin.snakemake)("--version")
```